src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -72,15 +75,13 @@ const Input = styled.input<{
   left: 0;
   right: 0;
   bottom: 0;
+  z-index: 97;
```

Comment:
Best variabelen voor aanmaken in stapjes van 10 of 100 om te voorkomen dat we een wildgroei aan lagen krijgen.